api/search.go

```diff
@@ -28,7 +31,11 @@ const defaultContentTypes string = "bulletin," +
 	"static_methodology_download," +
 	"static_page," +
 	"static_qmi," +
-	"timeseries"
+	"timeseries," +
+	"timeseries_dataset," +
+	"product_page," +
+	"release," +
+	"reference_tables"
```

Comment:
Is this a duplicate? It might be worth reordering this list in alphabetical order making it easier to spot potential duplicates